usaspending_api/api_contracts/contracts/subawards/Subawards.md

```diff
@@ -0,0 +1,56 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Subawards
+
+These endpoints are used to power USAspending.gov's subaward listings.
+
+# Subaward List
+
+This endpoint returns a filtered set of subawards.
+
+## Subawards [/api/v2/subawards/]
+
+This endpoint returns a list of data that is associated with the award profile page.
+
+### Subawards [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + page (required, number)
+            + Default: 1
+        + limit (optional, number)
+            + Default: 10
+        + filters (required, FilterObject)
+        + order (required, enum[string], fixed-type)
+            + Members
+                + asc
+                + desc 
+            + Default: desc
```

Comment:
For reasons I don't fully understand, Dredd complains if a Default value is also listed in the Members list.
```
duplicit value in enumeration on lines 25-28
```

Reply:
Nope.  You're right.  I had added a value for testing.  It was the value that was causing that issue not the default.  My apologies.